### DISCUSSION TITLE
Expose the lib for testing purpose

### DISCUSSION THREAD

**Discussion by soywod** [stalwartlabs/jmap-server#31] [Ideas]
Would it make sense to expose the lib (on `crates.io`) for testing purpose? For my integration tests I manually spawn a [GreenMail](https://github.com/greenmail-mail-test/greenmail) instance via docker, which works well but is not so convenient (nor portable). Same question for the SMTP server.

**Comment by mdecimus** [comment]
Thanks for the suggestion. There is a code refactoring coming for the JMAP and IMAP servers so I'll keep this in mind.

**Comment by soywod** [comment]
Awesome, thank you!
